### PR TITLE
Motionblur

### PR DESCRIPTION
**Pull request by arunraja-hub**
Completed.

### CODE REVIEW

**Comment by arunraja-hub** [comment]
`AssertionError: Bounding boxes for motionblur do not match` despite making no changes to bbox
Possible issue with snapshots, @ian-datature will look into this

File: src/discolight/augmentations/motionblur.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import cv2
+from discolight.params.params import Params
+from .augmentation.types import Augmentation
+from .decorators.accepts_probs import accepts_probs
+
+
+@accepts_probs
+class motionblur(Augmentation):
```
Comment (ian-datature) [inline src/discolight/augmentations/motionblur.py]: Class names should start with a capital letter and be in camel case (e.g., `MotionBlur`)

File: src/discolight/augmentations/motionblur.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    """Add motionblur to a given image."""
```
Comment (ian-datature) [inline src/discolight/augmentations/motionblur.py:27]:
```suggestion
    """Add motion blur to a given image."""
```

File: src/discolight/augmentations/motionblur.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    def __init__(self, kernel_size, direction):
+        super().__init__()
+
+        self.kernel_size = kernel_size
+        self.direction = direction
+
+    @staticmethod
+    def params():
+        """Return a Params object describing constructor parameters."""
+        return Params().add(
```
Comment (ian-datature) [inline src/discolight/augmentations/motionblur.py:44]: Please pull the latest changes from `master` and update the `Params` accordingly. If there is a limit on the `kernel_size` you should use a `BoundedNumber` type. The `direction` parameter should probably be an enum. You can see in the README how to define and use these types.

**Review by ian-datature — CHANGES_REQUESTED** [review]
I've made a few comments / suggestions. I also fixed the snapshots issue in #6. When this is merged you should pull from `master`,  regenerate the snapshots, and push the updated snapshot files as a commit to this pull request. Thanks!

**Review by ian-datature — APPROVED** [review]

**Review by gkeechin — APPROVED** [review]

**Pull request merged**